Assignment-1-6/emailValidate.php

```diff
@@ -0,0 +1,32 @@
+<?php
+    /**
+     * Validate the Email
+     * @param [string] $email
```

Comment:
Keep one blank line after the function description

Reply:
solved.